### PR TITLE
Add initial screens and navigation

### PR DESCRIPTION
**Pull request by jleach**
Add place holder screens, views and navigation. No functionality added as of yet.

### CODE REVIEW

File: SecureImage/UI/Albums/AlbumsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+// Created by Jason Leach on 2017-12-13.
+//
+
+import UIKit
+
+class AlbumsViewController: UIViewController {
+
+    @IBOutlet weak var tableView: UITableView!
+    
+    private let albumDetailsSequeID: String = "ShowAlbumDetailsSegue"
```
Comment (AmirShayegh) [inline SecureImage/UI/Albums/AlbumsViewController.swift]: albumDetailsSequeID should be albumDetailsSegueID

File: SecureImage/UI/Albums/AlbumsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        createFirstAlbumView.widthAnchor.constraint(equalToConstant: 250).isActive = true
+        createFirstAlbumView.heightAnchor.constraint(equalToConstant: 100).isActive = true
+        createFirstAlbumView.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        createFirstAlbumView.centerYAnchor.constraint(equalTo: view.centerYAnchor).isActive = true
+        
+        createFirstAlbumView.onCreateFirstAlbumTouched = { [weak self] in
+            // Create a new album in the db
+            // Pass album ID to Album Details VC
+
+            guard let sequeID = self?.albumDetailsSequeID else {
```
Comment (AmirShayegh) [inline SecureImage/UI/Albums/AlbumsViewController.swift]: sequeID should be segueID

File: SecureImage/UI/Albums/AlbumsViewController.swift (hunk 1 above)
Comment (jleach) [reply SecureImage/UI/Albums/AlbumsViewController.swift]: Fixed

File: SecureImage/UI/Albums/AlbumsViewController.swift (hunk 2 above)
Comment (jleach) [reply SecureImage/UI/Albums/AlbumsViewController.swift]: Fixed

**Review by AmirShayegh — APPROVED** [review]

**Pull request merged**